Language/Generics/Superbounded_types/class_A03_t03.dart

```diff
@@ -7,9 +7,8 @@
 ///   ...
 /// - `T` is an immediate subterm of a redirecting factory constructor signature
 ///
-/// @description Checks that it is a compile-time error if a parameterized
-/// super-bounded type `T` is used in a redirecting factory constructor
-/// signature
+/// @description Checks that it is a compile-time error if a super-bounded type
+/// is an immediate subterm of a redirecting factory constructor signature
```

Comment:
Perhaps just delete this test? I don't see any super-bounded types anywhere, we just have the (irrelevant) lack of subtyping.

Reply:
Deleted.